packages/salesforcedx-vscode-apex/src/apexLanguageClient.ts

```diff
@@ -21,4 +31,62 @@ export class ApexLanguageClient extends LanguageClient {
   public get errorHandler(): ApexErrorHandler | undefined {
     return this._errorHandler;
   }
+
+  public async stop(): Promise<void> {
+    await super.stop();
+  }
+
+  public async showOrphanedProcessesDialog(orphanedProcesses: ProcessDetail[]) {
```

Comment:
This method seems to mix the concern of showing a vscode dialog and the language client together.  What do you think about adding a `src/vscode/dialogService` to handle the logic of showing the dialog/channel info.  The Language client doesn't seem like it should be in the business of showing dialogs.

Reply:
Seems reasonable. I'll refactor UI related code out of language client

Reply:
thanks!  changes look good 👍